debian/stretch/foreman/rules

```diff
@@ -25,3 +25,9 @@ build:
 	/bin/sed -ri '1sXenv rubyXenv foreman-rubyX' bin/*
 	/bin/sed -ri 's~^BUNDLER_CMD=""~BUNDLER_CMD="/usr/bin/foreman-ruby /usr/bin/bundle exec"~' script/foreman-rake
 	/bin/mkdir -p .ssh
+	# Ensure backwards compatiblity by having dynflowd aliased to foreman-tasks
+	/bin/cp bin/dynflow-executor /usr/sbin/dynflowd
+	/bin/ln -s /usr/sbin/dynflowd /usr/sbin/ruby-foreman-tasks
+
+%:
+	dh --with=systemd
```

Comment:
This or `dh $@ --with=systemd` are failing with:

```
07:44:13 dh --with=systemd
07:44:13 dh: Compatibility levels before 9 are deprecated (level 7 in use)
07:44:13 dh: specify a sequence to run
07:44:13 debian/rules:33: recipe for target 'common-install-prehook-indep' failed
```

Any idea @mmoll ? @GregSutcliffe ?